CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# dbt_hubspot_source v0.6.3
+## Fixes
+- Fixes a bug in the models `stg_hubspot__engagement_meeting.sql` and `stg_hubspot__engagement_meeting_tmp.sql` where the `fivetran_utils.enabled_vars` macro was referencing the wrong variable (`hubspot_engagement_email_enabled`) from the vars list in the `dbt_project.yml`. Also updates `src_hubspot.yml` variable to `hubspot_engagement_meeting_enabled`. This was preventing users from disabling these enagagement_meeting models in their projects. 
```

Comment:
Can you also provide the link to this PR in the CHANGELOG. This just ensures that if people come to the CHANGELOG, they can view the PR where this change was applied.

```suggestion
- Fixes a bug in the models `stg_hubspot__engagement_meeting.sql` and `stg_hubspot__engagement_meeting_tmp.sql` where the `fivetran_utils.enabled_vars` macro was referencing the wrong variable (`hubspot_engagement_email_enabled`) from the vars list in the `dbt_project.yml`. Also updates `src_hubspot.yml` variable to `hubspot_engagement_meeting_enabled`. This was preventing users from disabling these enagagement_meeting models in their projects. ([#85](https://github.com/fivetran/dbt_hubspot_source/pull/85))
```

Reply:
Can you also add a Contributor section to make sure the customer (in this case @fivetran-seanodriscoll) is credited. 

See a previous CHANGELOG entry on how we give this credit.

Reply:
Final note, can you also break the `Also` part as a new bullet. Just to make the CHANGELOG a bit easier to digest as smaller pieces.